src/cmap/auth/mongo_credentials.ts

```diff
@@ -45,7 +45,9 @@ export const DEFAULT_ALLOWED_HOSTS = [
   '*.mongodbgov.net',
   'localhost',
   '127.0.0.1',
-  '::1'
+  '::1',
+  '*.mongo.com',
+  '*.mongodbgov.net'
```

Comment:
*.mongodbgov.net is already present on line 45. (The spec PR also duplicated it)